.github/workflows/main.yml

```
@@ -0,0 +1,55 @@
+name: build CI
+
+on: [push, pull_request]
+
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, '[skip ci]')"
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.6", "3.7", "3.8"]
+
+    steps:
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Upgrade pip And setuptools
+        run: python -m pip install -U pip setuptools pipenv
+
+      - name: Checkout workspace
+        uses: actions/checkout@v2
+
+      - name: Install Poetry
+        shell: bash
+        run: |
+          curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+          echo "::set-env name=PATH::$HOME/.poetry/bin:$PATH"
+          source $HOME/.poetry/env
+          poetry --version
+
+      - name: Configure poetry
+        shell: bash
+        run: poetry config virtualenvs.in-project false
+
+      - name: Install Python Dependencies
+        run: poetry install
```

I think this needs to be

```suggestion
        run: poetry install -E testing
```

---

Done.